packages/vite/src/node/plugin.ts

```diff
@@ -323,6 +323,20 @@ export interface Plugin<A = any> extends RollupPlugin<A> {
   >
 }
 
+export interface CustomPluginOptionsVite {
```

Comment:
I moved it here because it seemed `CustomPluginOptions` was changed to a `type` from an `interface`.

Reply:
I think this might probably be unintentional from Rollup side due to https://github.com/rollup/rollup/pull/5591 :thinking:

Reply:
Made a PR here 👍 
https://github.com/rollup/rollup/pull/5850